packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -113,6 +114,10 @@ class BuildInfo {
   /// rerun tasks.
   final String performanceMeasurementFile;
 
+  /// If provided, an output file where a v8-style heapsnapshot will be written for size
+  /// profiling.
+  final String analyzeSize;
```

Comment:
perhaps we should store an output file for each platform - see https://github.com/flutter/flutter/issues/61629

Reply:
If we need to do multiple platforms, we'll need to create multiple BuildInfos anyway